bundles/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/StringUtils.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.lutron.internal;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+/**
+ * Supply some string utility methods formerly provided by org.apache.commons.lang.StringUtils.
+ *
+ * @author Bob Adair - Initial contribution
+ *
+ */
+@NonNullByDefault
+public class StringUtils {
+
+    public static boolean equals(@Nullable String s1, @Nullable String s2) {
+        if (s1 == null && s2 == null) {
+            return true;
+        } else if (s1 == null || s2 == null) {
+            return false;
+        } else {
+            return s1.equals(s2);
+        }
+    }
```

Comment:
This can be replaced with `Objects.equals()`

Reply:
LOL. So it can! Somehow that didn't even occur to me. :-)